Applications/meson.build

```diff
@@ -2,6 +2,7 @@ nntr_app_resdir = nntrainer_resdir / 'app'
 run_command('mkdir', '-p', nntr_app_resdir)
 
 subdir('utils/jni')
+subdir('utils/datagen/cifar')
```

Comment:
Please make a another meson.build file in utils

Reply:
Fix